list.go

```diff
@@ -0,0 +1,90 @@
+// +build linux
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/codegangsta/cli"
+)
+
+var listCommand = cli.Command{
+	Name:  "list",
+	Usage: "lists containers started by runc with the given root",
+	Action: func(context *cli.Context) {
+
+		// preload the container factory
+		if factory == nil {
+			err := factoryPreload(context)
+			if err != nil {
+				logrus.Fatal(err)
+				return
+			}
+		}
+
+		// get the list of containers
+		root := context.GlobalString("root")
+		absRoot, err := filepath.Abs(root)
+		if err != nil {
+			logrus.Fatal(err)
+			return
+		}
+		list, err := ioutil.ReadDir(absRoot)
+
+		w := tabwriter.NewWriter(os.Stdout, 12, 1, 3, ' ', 0)
+		fmt.Fprint(w, "ID\tPID\tSTATUS\tSTARTED\tBUNDLE\tROOTFS\n")
```

Comment:
Now I see what your question was about... :) 

I'm just reporting the currently "stored" values for the containers and their state as stored on disk.  I'm not validating whether or not the stored values accurately reflect the actual state of said containers.  

To address your point/issue I could make a call to  `container.refreshstate()` hmm... Let me sleep on that and see what the others think about it.  I suppose at issue is should the runc list command be true to the stored state on disk or should I run refresh on those containers to fact check the state at the point of reporting it... Hmm :-)